src/components/APIDoc/Operation.tsx

```diff
@@ -63,11 +96,15 @@ const OperationContent: React.FunctionComponent<OperationProps> = ({verb, path,
     path: path,
     params: parameters,
     requestBody: operation.requestBody,
-    responses:  operation.responses,
+    responses: operation.responses,
     document: document,
-  }
+  };
   // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think this one is no longer needed. Looks like it got pulled because of the formatting changes

Reply:
> fixed, and yes i used prettify!

Not for the purpose of this PR.

Do you know if there is any eslint that confirms (or is close by) to prettify? It would be great to have those 2 aligned - maybe we can even have a prettify check as part of the ci. Maybe we can create a JIRA for this.

It makes it easier to review code without having to deal with formatting